clusterloader2/testing/load/config.yaml

```diff
@@ -428,45 +350,15 @@ steps:
       duration: {{$SLEEP_AFTER_EXEC_DURATION}}
 {{end}}
 
-{{if not $EXIT_AFTER_EXEC}}
+- module:
```

Comment:
This merges some parts that were executed before and after ExecCommand so it changes test behaviour. Could we make something similar that we did in reconcile-module and have two actions "create" and "delete" so we wouldn't modify test behaviour?

Reply:
Done